crates/rust-analyzer/src/config.rs

```diff
@@ -313,8 +313,10 @@ config_data! {
         /// Whether to show keyword hover popups. Only applies when
         /// `#rust-analyzer.hover.documentation.enable#` is set.
         hover_documentation_keywords_enable: bool  = "true",
-        /// Use markdown syntax for links in hover.
+        /// Use markdown syntax for links on hover.
         hover_links_enable: bool = "true",
+        /// Whether to show memory layout data on hover.
+        hover_memory_layout_enable: bool = "true",
```

Comment:
```suggestion
        hover_memoryLayout_enable: bool = "true",
```

Let's treat this as one thing

Reply:
Makes sense, thank you. Fixed it.